beetsplug/lyrics.py

```diff
@@ -459,12 +524,16 @@ def __init__(self):
             'auto': True,
             'google_API_key': None,
             'google_engine_ID': u'009217259823014548361:lndtuqkycfu',
+            'genius_api_key':
+                "Ryq93pUGm8bM6eUWwD_M3NOFFDAtp2yEE7W"
+                "76V-uFL5jks5dNvcGCdarqFjDhP9c",
             'fallback': None,
             'force': False,
             'sources': self.SOURCES,
         })
         self.config['google_API_key'].redact = True
         self.config['google_engine_ID'].redact = True
+        self.config['genius_api_key'].redact = True
```

Comment:
Let's leave it, just to avoid accidentally exposing anyone's private API key if they opt to use a custom one.